package.json

```
@@ -70,13 +71,49 @@
         "command": "runme.resetEnv",
         "category": "Runme",
         "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.resetEnv",
+        "category": "Runme",
+        "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.openAsRunmeNotebook",
+        "category": "Runme",
+        "title": "Open markdown as Runme",
+        "icon": {
+          "light": "assets/runme-logo-light.svg",
+          "dark": "assets/runme-logo-dark.svg"
+        }
+      },
+      {
+        "command": "runme.openSplitViewAsMarkdownText",
+        "category": "Runme",
+        "title": "Open Runme notebook in text editor",
+        "icon": "$(markdown)"
       }
     ],
     "menus": {
       "commandPalette": [
         {
           "command": "runme.openTerminal",
           "when": "false"
+        }, 
+        {
+          "command": "runme.openAsRunmeNotebook",
+          "when": "false"
+        }
```

Should both commands be added to the command palette?

```suggestion
        }, 
        {
          "command": "runme.openAsRunmeNotebook",
          "when": "false"
        }, 
        {
          "command": "runme.openSplitViewAsMarkdownText",
          "when": "false"
        }
```